### DISCUSSION TITLE
XMLBuilder cdata text

### DISCUSSION THREAD

**Discussion by sberan1** [NaturalIntelligence/fast-xml-parser#638] [Q&A]
Hello, 

I want to parse a JSON to XML but I need to wrap some data in ![CDATA[]]. Is there any possibility in the builder? 

### Example 
```json
"Person": {
   "Id": 1249,
   "Name": "Carl"
}
```

```xml
<Person> 
   <Id>1249</Id>
   <Name><![CDATA[Carl]]></Name>
</Person> 
```
  
  Is there any way to call the builder.build() function and specify the fields where I want the CDATA wrap so people are able to use all the characters?
  
  I tried searching for a resolution but couldn't find any similar issue being open or any discussion about this topic.
  
  Thank you in advance for any answer

**Comment by amitguptagwl** [comment]
Please check https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/3.XMLBuilder.md#cdatapropname
